pyproject.toml

```diff
@@ -17,7 +17,7 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = "^3.7"
-httpx = "^0.7.4"
+httpx = "^0.11.1"
```

Comment:
Please review https://github.com/encode/httpx/blob/master/CHANGELOG.md
They post `API` changes, and some look pretty drastic...